CharacterCreation/02_Classes.txt

```diff
@@ -1,25 +1,44 @@
-Core classes:
-/*====================================================
- * Your character starts at level 0. At level 0 you have skills, but no class, 
- * class feats or regular feats.
- * At level 1 you get to select a class, and receive the class passives. 
- * However, you do not select a class feat until level 5. Then, at levels 10, 15, and 20
- * you can gain another class feat.
- *
- * For any listed class, if it says "Gain X Skill at 10" that means you get
- * to add the X skill to your skill list. For more on Skills, see 
- * RP / Using Skills Section of Basic Rules.txt. 
- *==================================================*/
+==============================
+Core Classes
+==============================
+
+	Your character starts at level 0. At level 0 you have skills, but no class, 
+class feats or regular feats. At level 1 you get to select a class, and receive 
+the class passives. However, you do not select a class feat until level 5. Then, 
+at levels 10, 15, and 20 you can gain another class feat.
+
+	For any listed class, if it says "Gain X Skill at 10" that means you get to 
+add the X skill to your skill list. For more on Skills, see "RP / Using Skills" 
+Section of Basic Rules.txt. 
+
 
 =====   SOLDIER   =====
-You have the blood of a warrior flowing through you. Your sole 
-purpose is to fight until the threat is neutralized.
+
+	You have the blood of a warrior flowing through you. Your sole purpose is to 
+fight until the threat is neutralized.
+
 Class Primary Attribute: Strength
-Passives: Gain Skill Weapon Use[X] at 10, Gain a +15 to Rolls for initiative. (You go first more often).
-Weapon Proficiencies: Pistols, Submachine Guns, Shotguns, Carbines, Assault Rifles, Battle Rifles.
-Armor Proficiencies: Light, Medium, Medium-Heavy.
 Starting Wealth: $3770 + 100*Luck
-Class Feats:
+
+
+Passives: 
+* Gain Skill Weapon Use[X] at 10 
+* Gain a +15 to Rolls for initiative. (You go first more often).
+
+Weapon Proficiencies: 
+* Pistols 
+* Submachine Guns 
+* Shotguns 
+* Carbines
+* Assault Rifles
+* Battle Rifles.
+
+Armor Proficiencies: 
+* Light
+* Medium
+* Medium-Heavy
```

Comment:
Idk Michael. 16 lines to display 3 lines of information. I really don't like all the space; I think it will make it much harder to scroll through the doc to compare classes.

Reply:
1) This is a public repo, get my name out of there. Get it out now. 
2) If you want to address me, @ me. 
3) I hear you, long files are bad. But by breaking up the information, it actually becomes easier to skim to the part of the info that you're looking for thereby making it easier to find and work on what you're trying to do. For players, they'll be using the CXDocs page which will render each of these into Divs which can then be aligned to be side-by-side. Better yet, if the computer can read and differentiate each class, we may be able to actually add a side-by-side tool. But in order to add that quality of life, we need CXDocs to be able to read what we're writing rather than needing a human-in-the-loop to interpret each change.

Reply:
I concur that spacing the information vertically will make comparison faster despite marginally increased scrolling time.